coalib/bearlib/aspects/Smell.py

```diff
@@ -57,7 +56,7 @@ class docs:
         * Feature envy
         etc ...
         """
-        example_language = 'English'
+        example_language = 'rst'
```

Comment:
My previous comment was perhaps a little confusing.  This should not use prose --- it should be sample code, demonstrating a smell, like the other Root subaspects.